src/sqlfluff/dialects/dialect_mysql.py

```diff
@@ -1175,6 +1177,77 @@ class AlterTableStatementSegment(BaseSegment):
     )
 
 
+class WithCheckOptionSegment(BaseSegment):
+    """WITH [CASCADED | LOCAL] CHECK OPTION for CREATE/ALTER View Syntax.
+
+    As specified in https://mariadb.com/kb/en/alter-view/
```

Comment:
Why are we linking to mariadb docs instead of mysql docs?

Reply:
Yes, I just added the link to MySQL docs

Reply:
Should we just have the MySQL links instead of the mariadb links?

Reply:
Given that MySQL and MariaDB use almost the same dialect, I thought it should be good to know that this will support them both. What do you think?

Reply:
I think it's slightly cleaner to just put MySQL only, like the rest of the file has it.

Reply:
done